Mathlib/Order/WellFoundedSet.lean

```diff
@@ -679,6 +679,25 @@ namespace Set.PartiallyWellOrderedOn
 
 variable {r : α → α → Prop}
 
+theorem bddAbove_preimage (s : Set α) (hs : s.PartiallyWellOrderedOn r) (f : ℕ → α)
```

Comment:
Would it make sense to have `s` and `f` implicit since they can be inferred from `hs` and `hf` respectively, or is this theorem likely to be applied where they'll have to be specified?
```suggestion
theorem bddAbove_preimage {s : Set α} (hs : s.PartiallyWellOrderedOn r) {f : ℕ → α}
```